crates/iota-sdk-types/src/digest.rs

```diff
@@ -224,4 +237,31 @@ mod tests {
         let d = s.parse::<Digest>().unwrap();
         assert_eq!(digest, d);
     }
+
+    #[test]
+    fn test_lexical_order() {
```

Comment:
Are these tests from the monorepo?

Reply:
no, I made it

Reply:
these are already covering those cases basically